setup.cfg

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2018 NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+[metadata]
+name = metomi-rose
+description = Rose, a framework for meteorological suites.
+author = 2012-2019 British Crown (Met Office) & Contributors
+author_email = metomi@metoffice.gov.uk
+url=https://metomi.github.io/rose/doc/html/index.html
+license = GPL
```

Comment:
This seems to have changed relative to the corresponding entry in the lines removed from the ``setup.py``. Is there a reason for this?

Reply:
I don't think you are actually meant to use hyperlinks. It certainly doesn't work in the CFG file - the build gives an error message. This is consistent with Cylc. I can put it back as was: What do you think?

Reply:
Sure, I just wanted to understand the reason this line had changed, in case there was a mistake in the translation across from the ``setup.py``. What you have now sounds right, so I'll mark this as resolved.